packages/contracts-bedrock/deploy/016-ProxyAdminL1ERC721BridgeProxy.ts

```diff
@@ -0,0 +1,39 @@
+import assert from 'assert'
+
```

Comment:
We could prob delete this file and just set the owner of the erc721bridge proxy to the proxy admin at deploy time